projects/core/directives/dropdown/dropdown-position.directive.ts

```diff
@@ -44,6 +44,10 @@ export class TuiDropdownPositionDirective extends TuiPositionAccessor {
             top: hostRect.top - offset - height,
             bottom: hostRect.bottom + offset,
             right,
+            center:
+                hostRect.left + width < viewport.right - offset
```

Comment:
Shouldn't it be `hostRect.left + hostRect.width / 2 + width / 2 < viewport.right - offset`?

Reply:
yes, I guess so